src/components/layout/Finishable/__specs__/FinishableContainer.spec.js

```diff
@@ -102,16 +131,18 @@ describe('components | FinishableContainer', () => {
         const props = mapStateToProps(state, ownProps)
 
         // then
-        expect(props).toStrictEqual({
-          offerCanBeBooked: false,
-        })
+        expect(props.offerCanBeBooked).toBe(false)
       })
 
       it('should return true when offer is bookable', () => {
         // given
+        getIsBooked.mockReturnValue(false)
+
         const state = {
           data: {
+            bookings: [{ id: 'A1', stockId: 'B1' }],
             offers: [{ id: 'A1', isBookable: true }],
```

Comment:
```suggestion
            offers: [{ id: 'C1', isBookable: true }],
```
Même remarque qu'au dessus pour penser à le modifier dans les `ownProps` qui suivent.

Reply:
👍